Mathlib/Combinatorics/SimpleGraph/Metric.lean

```diff
@@ -43,105 +41,190 @@ variable {V : Type*} (G : SimpleGraph V)
 
 /-! ## Metric -/
 
+section edist
+
+/--
+The extended distance between two vertices is the length of the shortest walk between them.
+It is `⊤` if no such walk exists.
+-/
+noncomputable def edist (u v : V) : ℕ∞ :=
+  ⨅ w : G.Walk u v, w.length
+
+variable {G} {u v w : V}
+
+protected theorem Reachable.exists_walk_of_edist (hr : G.Reachable u v) :
```

Comment:
The dist theorems don't seem to follow the mathlib naming convention. Would you mind updating them and these edist theorems? I'm thinking this could be `Reachable.exists_walk_length_eq_edist` (and similar for `Connected` and the pre-existing dist versions).

Reply:
Thank you.  I've updated the naming of the lemmas that don't follow the naming convention, but I find naming lemmas confusing as there are lots of exceptions and subtleties.  Please let me know if the names are good now or if any need changing.

Reply:
Sorry, I meant to say *just* these four theorems needed changing. Could you revert your last commit please?

Reply:
Sure!